doc/modules/preprocessing.rst

```diff
@@ -235,6 +236,8 @@ more robust estimates for the center and range of your data.
   To address this issue you can use :class:`~sklearn.decomposition.PCA` with
   ``whiten=True`` to further remove the linear correlation across features.
 
+|details-end|
```

Comment:
This closing of the dropdown should be above, it should not contain the `Scaling vs Whitening` topic (which could itself be another dropdown).